backend/src/Entity/OrderLogsEntity.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\OrderLogsEntityRepository;
+use Doctrine\ORM\Mapping as ORM;
+use Gedmo\Mapping\Annotation as Gedmo;
+
+#[ORM\Entity(repositoryClass: OrderLogsEntityRepository::class)]
+class OrderLogsEntity
+{
+    #[ORM\Id]
+    #[ORM\GeneratedValue]
+    #[ORM\Column(type: 'integer')]
+    private $id;
+
+    #[ORM\ManyToOne(targetEntity: OrderEntity::class, inversedBy: 'OrderLogsEntity')]
+    #[ORM\JoinColumn(nullable: false)]
+    private $orderId;
+
+    #[ORM\ManyToOne(targetEntity: CaptainEntity::class, inversedBy: 'OrderLogsEntity')]
+    private $captainProfile;
```

Comment:
if this not null, how you will be able to insert the first status of order: "order created" ?